SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -277,6 +329,13 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 			: SpatialOSLaunchConfig.FilePath;
 	}
 
+	FORCEINLINE FString GetSpatialOSConfig()
```

Comment:
Why not use `FSpatialGDKServicesModule::GetSpatialOSDirectory()` :D

Reply:
It's actually used - this seems to be a leftover, removing it 🗑.